sharding-metrics/sharding-metrics-facade/src/test/java/org/apache/shardingsphere/metrics/facade/MetricsTrackerFacadeTest.java

```diff
@@ -67,16 +70,28 @@ public void gaugeDec() {
     
     @Test
     public void histogram() {
+        MetricsConfiguration metricsConfiguration = new MetricsConfiguration("fixture", null, null, null);
+        metricsTrackerFacade.init(metricsConfiguration);
+        assertEquals(metricsTrackerFacade.getMetricsTrackerManager().getClass(), MetricsTrackerManagerFixture2.class);
         HistogramMetricsTrackerDelegate delegate = metricsTrackerFacade.histogramStartTimer("request");
-        assertEquals(delegate.getClass(), NoneHistogramMetricsTrackerDelegate.class);
         metricsTrackerFacade.histogramObserveDuration(delegate);
+        assertEquals(delegate.getClass(), NoneHistogramMetricsTrackerDelegate.class);
```

Comment:
Please use assertThat instead of assertEquals

Reply:
ok